xblock/runtime.py

```diff
@@ -54,22 +54,18 @@ def __new__(cls, scope, user_id, block_scope_id, field_name, block_family='xbloc
     @abstractmethod
     def get(self, key):
         """Reads the value of the given `key` from storage."""
```

Comment:
I think you can disable the pass warning instead of removing.

Reply:
Should I go on and re-add all the `pass`?